src/EventStore.Core/TransactionLog/Chunks/TFChunk/TFChunk.cs

```diff
@@ -1414,4 +1415,15 @@ internal readonly ref struct Slot(ReaderWorkItem[] array, int index) {
 			public ref ReaderWorkItem ValueRef => ref UnsafeGetElement(array, index);
 		}
 	}
+
+	public ChunkInfo GetChunkInfo() {
```

Comment:
Why not just a property? Especially if `ChunkInfo` is **struct**.

Reply:
Fixed (i considered that initially and used a method instead so that it doesn't look like the "ChunkInfo" information is static - it can change when the chunk is completed. but i think it's fine to use a property as well)

Reply:
ChunkInfo is a class at the moment though. since it can change maybe the property should be called `CurrentInfo`